src/backend/access.rs

```diff
@@ -61,10 +61,12 @@ impl AccessResponse {
     }
 }
 
+#[allow(clippy::too_many_arguments)]
 #[async_trait]
 pub trait AccessImpl: RequestImpl {
     async fn access_dialog(
```

Comment:
```suggestion
#[async_trait]
pub trait AccessImpl: RequestImpl {
    #[allow(clippy::too_many_arguments)]
    async fn access_dialog(
```
instead

Reply:
That's where I wanted to put too, but by mistake put it over the trait. Thanks.